controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -218,15 +218,22 @@ def _mock_cli_object(cli_object):
         return Mock(as_single_element=cli_object)
 
     @staticmethod
-    def _get_cli_volume():
+    def _get_cli_volume(enable_deduplicated_copy=True):
```

Comment:
I would remove the `enable_`

Reply:
done

Reply:
I wouldn't. there is already `deduplicated_copy `parameter in the method

Reply:
then consider replacing it with `with_`

Reply:
done